cli/cmd/project/delete.go

```diff
@@ -22,17 +25,48 @@ func DeleteCmd(cfg *config.Config) *cobra.Command {
 			}
 			defer client.Close()
 
+			// Get the new project name from user if not provided in the flag
+			if !cmd.Flags().Changed("project") {
```

Comment:
This will not prompt for a name if empty name is given by the user. Is that expected ?

Reply:
@begelundmuller Your thoughts on this? We can check for `==""` for projects but not for `orgs` as we set the default org for projects.

Also we have checks for `value length` for empty in API validate. So any suggestions on this?